.github/workflows/ci.yml

```diff
@@ -59,6 +62,11 @@ jobs:
           repository: rocm/jax
           ref: rocm-jaxlib-v0.7.1
           path: jax
+      - name: Apply patches to rocm/jax test repo
+        run: |
+          pushd jax
+          git apply ../ci/patches/*
+          popd
```

Comment:
Shouldnt this be done automatically by bazel  ? Why do we need to do this manually ?

Reply:
Because these get applied to the rocm/jax that we use for test cases, not the jax-ml/jax we use for wheel builds. Maybe it'd be better to carry this as a change in the rocm/jax repo itself rather than a patch? I'd really rather not carry anything in that repo though, and it's unfortunate that we have to keep jaxlib changes in there right now.